scripts/release_info.py

```diff
@@ -195,7 +191,6 @@ def _check_storage_models(current_release):
 
     Returns:
         (list): The changed files (if any)
```

Comment:
Same -- list(str).

Also there's no period at the end. Shouldn't the docstring check catch that?

Reply:
@seanlip Actually the check looks for any character that is in the `ALLOWED_TERMINATING_PUNCTUATIONS` and ')' is in that list. It was there before so I did not change it. Should I change this so as to it would only look for a period?

Reply:
Ah, thanks, that's interesting. In which case would ")" be a valid terminating character -- what are some examples in the current codebase? If there aren't any then yep, changing it is fine, but I want to make sure first that we aren't causing unwanted regressions by doing so.

Reply:
@seanlip I am not sure why we need other characters in `ALLOWED_TERMINATING_PUNCTUATIONS`. Some TODO end with '?'  and I think the rest can be made to end with '.' . Following are some examples:
* https://github.com/oppia/oppia/blob/develop/core/jobs.py#L611 -> This is a TODO
* https://github.com/oppia/oppia/blob/develop/core/storage/feedback/gae_models.py#L52
* https://github.com/oppia/oppia/blob/develop/core/domain/summary_services.py#L79

Reply:
Thanks! Those are interesting examples, thanks for finding them.

I don't have a strong opinion on this, so I will defer to you and @apb7 to set a standard for the codebase. I'm happy with whatever you decide.

Reply:
Hi @seanlip after discussing with @apb7 we decided that the docstrings can end with any character present in `ALLOWED_TERMINATING_PUNCTUATIONS`